doc/content/source/materials/NEMLStress.md

```diff
@@ -8,20 +8,27 @@ This material model provides an interface to the constitutive models for structu
 
 NEML permits the definition of complex material constitutive models, the details of which are defined in an XML file that is independent of the MOOSE input file. The required inputs for this model on the MOOSE side simply provide the location of the XML file, which contains a database of potentially multiple material models, and the name of the model within that database to be used.  This model computes the stress, elasticity tensor, and material Jacobian.
 
-Using this model requires that the NEML source code is present, and that the prerequisite libraries are available. NEML is provided as a submodule in BlackBear under the directory `blackbear/contrib/neml`. The up-to-date version of the NEML submodule is obtained using:
+Using this model requires that the NEML source code is present, and that the prerequisite libraries are available. 
+NEML is provided as a submodule in BlackBear under the directory `blackbear/contrib/neml`. 
+The up-to-date version of the NEML submodule is obtained using:
+
 ```
+cd blackbear/contrib/neml
```

Comment:
I think you can just do it from the root 'blackbear' directory, and I don't think it will actually work from `blackbear/contrib/neml` since that is inside the submodule.